test/Rounds-test.js

```diff
@@ -0,0 +1,27 @@
+const chai = require('chai');
+const expect = chai.expect;
+const spies = require('chai-spies')
+chai.use(spies);
+const Round = require('../lib/rounds.js');
+global.domUpdates = require('../lib/domUpdates.js');
+global.Game = require('../lib/game.js');
+global.Player = require('../lib/player.js');
+global.data = require('../lib/jeopardy-dataset.js');
+chai.spy.on(global.domUpdates, ['updateCategories'], () => true);
```

Comment:
Good setup of global variables and spies, and good use of your beforeEach block down below